runtime/doc/treesitter.txt

```diff
@@ -475,10 +497,10 @@ The following captures are linked by default to standard |group-name|s (use
 @comment                line and block comments
 @comment.documentation  comments documenting code
 
-@comment.error          error-type comments (e.g. `ERROR`, `FIXME`, `DEPRECATED:`)
+@comment.error          error-type comments (e.g. `ERROR`, `FIXME`)
 @comment.warning        warning-type comments (e.g. `WARNING:`, `FIX:`, `HACK:`)
 @comment.todo           todo-type comments (e.g. `TODO:`, `WIP:`, `FIXME:`)
-@comment.note           note-type comments (e.g. `NOTE:`, `INFO:`, `XXX`)
+@comment.note           note-type comments (e.g. `NOTE:`, `INFO:`, `XXX:`)
```

Comment:
Same (used in `comment`).

Reply:
Hmm I see, looking at CONTRIBUTING.md of nvim-treesitter the use of colon was intentional?

Reply:
Yep, that's why I wrote above that removing colons for consistency is a good idea here ;)

(The important information here is what sort of keyword belongs in which severity, which is clear(er) without colon as well.)

Reply:
Doesn't really matter in either way but I'd vote for the consistency (i.e. remove all the colons here) as this will look least confusing.